alembic/env.py

```diff
@@ -68,6 +68,41 @@ def include_object(object, name, type_, reflected, compare_to):
 # ... etc.
 
 
+# Operation to run before each migration...
+def drop_views(conn):
+    logger.info("Dropping views if they exist...")
+    global schema_name
+    conn.execute(text(f"DROP VIEW IF EXISTS {schema_name}.visit1"))
```

Comment:
Is there some way to reuse this text in the `run_migrations_offline` method?  It feels weird to have it twice.